submit-web/src/components/Submission/DocumentTable.tsx

```diff
@@ -71,92 +87,119 @@ export default function DocumentTable({
           </TableRow>
         </TableHead>
         <TableBody>
-          {documents?.map((document) => (
-            <TableRow
-              key={`row-${document.id}`}
-              component={Box}
-              sx={{ mb: 1, backgroundColor: BCDesignTokens.themeBlue10 }}
-            >
-              <TableCell
-                component="th"
-                scope="row"
-                colSpan={6}
-                sx={{
-                  borderTop: "2px solid #F2F2F2",
-                  borderBottom: "2px solid #F2F2F2",
-                  borderLeft: "2px solid #F2F2F2",
-                  borderTopLeftRadius: 5,
-                  borderBottomLeftRadius: 5,
-                  py: BCDesignTokens.layoutPaddingSmall,
-                }}
+          {sortedDocuments?.map((document) => (
+            <>
```

Comment:
it's not good to add this fragment. A map function should return React nodes with the prop key. This is not really an option. What are we trying to do here?